cluster/src/assembly/resources/conf/iotdb-cluster.properties

```diff
@@ -86,7 +86,7 @@ default_replica_num=3
 # so the leader will send logs(snapshot) to the follower,
 # NOTICE, it may cost minutes of time to send a snapshot,
 # so this parameter should be larger than the snapshot cost time.
-# catch_up_timeout_ms=300000
+# catch_up_timeout_ms=600000
```

Comment:
Hi, the catchUpTimeoutMS is not consistent with the iotdb-cluster.properites, in the ClusterConfig.java, the value is 60 seconds, while in the iotdb-cluster.properites, the value is 600 seconds, may be we can change to 300 seconds as the snapshot may take a long time, how do you think?

Reply:
ok